ui/app/components/clients/page/counts.hbs

```diff
@@ -121,9 +122,12 @@
       {{! CLIENT COUNT PAGE COMPONENTS RENDER HERE }}
       {{yield}}
 
-    {{else if (and (not @config.billingStartTimestamp) (not @startTimestamp))}}
-      {{! Empty state for no billing/license start date }}
-      <EmptyState @title={{this.versionText.title}} @message={{this.versionText.message}} />
+    {{else if (and this.version.isCommunity (not @startTimestamp))}}
+      {{! Empty state for community without start QP }}
```

Comment:
For someone who forgets context easily, could you define QP in the comment? might help future devs or those needing another ☕️